docs/src/componentDocs/ScoreCard/playground/PlaygroundPage.tsx

```diff
@@ -0,0 +1,249 @@
+
+const inputConfig: InputConfig = [
+    // Required Props
+    {
+        id: 'headerTitle',
+        type: 'string',
+        typeLabel: 'string',
+        description: 'The primary text',
+        required: true,
+        initialValue: 'Substation 3',
+        category: 'Required Props',
+    },
+
+    // Optional Props
+    {
+        id: 'actionItems',
+        type: 'boolean',
+        typeLabel: `IconSource`,
+        initialValue: false,
+        description: 'Display actionItems in the header',
+        required: false,
+        category: 'Optional Props',
+    },
+    {
+        id: 'badgeOffset',
+        type: 'number',
+        typeLabel: `number`,
+        description: 'Vertical offset for the badge component',
+        required: false,
+        initialValue: -40,
+        minValue: -50,
+        maxValue: 50,
+        valueStep: 2,
+        defaultValue: 0,
+        category: 'Optional Props',
+    },
+    {
+        id: 'headerBackgroundImage',
+        type: 'select',
+        typeLabel: `ImageSourcePropType`,
+        description: 'An image to display in the header',
+        initialValue: 'undefined',
+        defaultValue: 'undefined',
+        options: ['undefined', 'pattern'],
+        required: false,
+        category: 'Optional Props',
+    },
+    {
+        id: 'headerColor',
+        type: 'color',
+        typeLabel: 'string',
+        description: 'The color of the background background',
+        required: false,
+        initialValue: '',
+        category: 'Optional Props',
+    },
+    {
+        id: 'headerFontColor',
+        type: 'color',
+        typeLabel: 'string',
+        description: 'The color of the header text',
+        required: false,
+        initialValue: '',
+        defaultValue: '#ffffff',
+        category: 'Optional Props',
+    },
+    {
+        id: 'headerInfo',
+        type: 'string',
+        typeLabel: 'string',
+        description: 'The tertiary text',
+        required: true,
+        initialValue: '4 Devices',
+        category: 'Optional Props',
+    },
+    {
+        id: 'headerSubtitle',
+        type: 'string',
+        typeLabel: 'string',
+        description: 'The secondary text',
+        required: true,
+        initialValue: 'High Humidity Alarm',
+        category: 'Optional Props',
+    },
+
+    // Other Configuration
+    {
+        id: 'numberOfHeroes',
+        label: '# of Heroes',
+        type: 'number',
+        description: '# of heroes to show',
+        required: false,
+        initialValue: 1,
+        minValue: 0,
+        maxValue: 2,
+        valueStep: 1,
+        defaultValue: 1,
+        category: 'Additional Configuration',
+    },
+];
+
+const ScoreCardPreview: PreviewComponent = ({ data }) => {
+    const { headerBackgroundImage, actionItems, numberOfHeroes, ...rest } = data as unknown as ScoreCardProps & {
+        numberOfHeroes: number;
+    };
+    const theme = useExtendedTheme();
+    const heroes: JSX.Element[] = [
```

Comment:
update the heros to following
`const heroes: JSX.Element[] = [
        <Hero
            key={'hero2'}
            icon={{ family: 'brightlayer-ui', name: 'health' }}
            label={'Health'}
            iconSize={48}
            iconBackgroundColor={theme.colors.surface}
            ChannelValueProps={{ value: 54, units: '%' }}
        />,
        <Hero
            icon={{ family: 'brightlayer-ui', name: 'grade_a' }}
            label={'Grade'}
            iconSize={48}
            iconBackgroundColor={theme.colors.surface}
            ChannelValueProps={{ value: '98', units: '/100', unitSpace: 'hide' }}
        />,
    ];`